pyproject.toml

```diff
@@ -42,11 +40,21 @@ dependencies = [
 dev = [
     "pytest",
     "pytest-asyncio",
-    "nbval",  # For notebook testing
+    "nbval",                 # For notebook testing
     "black",
     "ruff",
     "types-requests",
     "types-setuptools",
+    "pre-commit",
+    "sphinx-autobuild",
```

Comment:
I wonder if we should add the sphinx deps separately in another subpackage? like `.docs` instead of `.dev`? I think .dev might get used in test setups a bunch and we don't want to add dependencies there.

Reply:
Could we add `sphinx-autobuild` here? https://github.com/meta-llama/llama-stack/blob/main/docs/requirements.txt

Reply:
@ashwinb  I'm fine with adding a new sub-dep section for building the website. Thanks for the suggestion.